Gruntfile.js

```diff
@@ -329,6 +329,8 @@ module.exports = function(grunt) {
 					[ WORKING_DIR + 'wp-includes/js/wp-embed.js' ]: [ './src/js/_enqueues/wp/embed.js' ],
 					[ WORKING_DIR + 'wp-includes/js/wp-emoji-loader.js' ]: [ './src/js/_enqueues/lib/emoji-loader.js' ],
 					[ WORKING_DIR + 'wp-includes/js/wp-emoji.js' ]: [ './src/js/_enqueues/wp/emoji.js' ],
+					[ WORKING_DIR + 'wp-includes/js/wp-image-mime-fallback-loader.js' ]: [ './src/js/_enqueues/wp/wp-image-mime-fallback-loader.js' ],
+					[ WORKING_DIR + 'wp-includes/js/wp-image-mime-fallback.js' ]: [ './src/js/_enqueues/wp/wp-image-mime-fallback.js' ],
```

Comment:
For the source file names on the right, let's omit the `wp-` prefix, similar to all the other ones above and below.

Reply:
Path updated.